strings/index.js

```diff
@@ -37,7 +37,13 @@ export const howLong = (str) => {
   I expect this function to join the strings a and b together
 */
 export const join = (a, b) => {
-  return a+b; // I expected concat(a, b) to also work, but it didnt??
+  if (typeof a === 'string' && typeof b === 'string') {
+    return a+b;
+  } else {
+    let convertA = String(a);
+    let convertB = String(b);
+    return convertA+convertB;
```

Comment:
Nice! You can also trim this down a shade further! If you try and cast a string to a string you just get a string, so you could do everything in the else statement and that would work for all cases! Make sense?

Reply:
man you're a sick teacher fuck me. you literally gave me the quicjkest way to the answer without telling me the answer gj man :D